app/frontend/src/test/utils.ts

```diff
@@ -1,13 +1,9 @@
 import { createAction } from "@reduxjs/toolkit";
 
 export function addDefaultUser() {
-  store.dispatch({
-    type: "auth/passwordLogin/fulfilled",
-    payload: {
-      token: "9hhiwXCaCzmTM6ZB7rfaJ0yZIlSsXTOQUZagihbcrAw=",
-      user: defaultUser,
-    },
-  });
+  window.localStorage.setItem("auth.authenticated", JSON.stringify(true));
+  window.localStorage.setItem("auth.jailed", JSON.stringify(false));
+  window.localStorage.setItem("auth.user", JSON.stringify(defaultUser));
```

Comment:
Is there a better way to make a "logged in user" default for tests?

Reply:
You can probably set the store directly to the user in a `beforeEach`? I pulled this out here rather than putting in `beforeEach` in setUpTests as I realised I didn't want this to be injected at the start of every tests, as some tests like the auth tests would need to start with no users first. Nothing stopping you from putting it in `beforeEach` in the test file that needs it though.

Reply:
There's no way to set the store directly now though- is it ok to add an optional parameter to useAuthStore for testing? Or another way?

Reply:
Since you had to make a mock implementation of local storage, couldn't you set it values directly against `store` exposed by `createLocalStorageMock`?

Reply:
I don't necessarily want it in every test. What I really meant was, is there a better way than pre-setting the mocked local storage. If you think it's fine, I'll leave as is.

Reply:
Probably not, I think it's fine to have it like this and it's why I got in a helper to minimise setup boilerplate, so for test files that need an authenticated user in all tests, you can put it in that file's `beforeEach`.